templates/CRM/Contribute/Form/Contribution/ThankYou.tpl

```diff
@@ -116,13 +116,9 @@
           {if !empty($auto_renew)} {* Auto-renew membership confirmation *}
             {crmRegion name="contribution-thankyou-recur-membership"}
               <br />
-              {if !$installments || $installments > 1}
```

Comment:
only thought here @eileenmcnaughton I wonder is if $installments is always being assigned to the template or not

Reply:
@seamuslee001 yeah - I'm confused - I'm thinking it maybe is not - but note that if it is NOT assigned there is still text rendered - it's just that there is

`   <strong>{ts 1=$frequency_unit}This membership will be renewed automatically every %1.{/ts}</strong>`

vs the per-unit version 

& it seems like the per unit version is generally preferred

Reply:
There is also a problem with 

`  <strong>{ts 1=$frequency_interval 2=$frequency_unit}This membership will be renewed automatically every %1 %2(s).{/ts}</strong>`

that seems to be pushed out - I don't know whether you can use the plural whatsit & have day translate to days in English - that seems like it has been left out of scope over a few iterations

Reply:
If I read this right, the `every %1 %2(s)` that is still there only affects when a limit is set on the number of instalments, which presumably is not used much.

Ideally it should use the same pattern as for unlimited instalments, with the if/elseif for day/week/month/year (but that's outside the scope of this PR).

Reply:
@mlutfy I pushed in an addition fix for that - if it looks good I suggest we merge this to the rc but only back port the original patch to 5.78

Reply:
@eileenmcnaughton  Looks good to me, thanks! I added merge-ready based on the feedback from Samuele on Gitlab (he had also worked on the initial fixes around this).